src/datasets/support/commercial-support.ts

```diff
@@ -22,6 +23,11 @@ type CommercialSupportType = {
  *  - Publicly available mention of OpenFeature support on the company's website or documentation
  */
 export const CommercialSupportList: CommercialSupportType[] = [
+  {
+    name: 'Kameleoon',
+    href: 'https://www.kameleoon.com/',
```

Comment:
Update with a link that mentions Kameleoon's support for OpenFeature.

Reply:
Hello @beeme1mr!

Hope you’re doing well!

I wanted to let you know that our team is currently working on the page, and I’ll get back to you with more details once it’s ready.

Thanks!

Reply:
Hello @beeme1mr!

Hope all is good!

Apologies for the delay, and thank you for your patience. Here is the updated link: [openfeature.dev PR #826](https://github.com/open-feature/openfeature.dev/pull/826).

Let me know if you have any questions or need further clarification.

Thanks!